.travis.yml

```diff
@@ -35,6 +35,9 @@ before_install:
   # remove due to system dependencies
   - sudo dpkg --purge --force-all libcryptsetup12
 
+install:
```

Comment:
You only need requests to be available for the Python linting tasks. So you should add requests to the list of things that are installed for those tasks (rather than installing for every task). You should also specify the version, which should be whatever is the default on Fedora 31.